contextualized/regression/trainers.py

```diff
@@ -10,35 +10,35 @@ class RegressionTrainer(pl.Trainer):
     Trains the contextualized.regression lightning_modules
     """
 
-    def predict_params(self, model, dataloader):
+    def predict_params(self, model, dataclass):
```

Comment:
This might be a good place to specify what set in the datamodule we're trying to predict from.